README.md

```diff
@@ -68,6 +74,14 @@ Save all data in JSON format:
 mkdelegation gen --json
 ```
 
+Generate delegations with specified private keys:
+```bash
+mkdelegation gen \
+--upload-service-private-key Mg... \
+--indexing-service-private-key Mg... \
+--storage-node-private-key Mg...
+```
+
```

Comment:
I'd like to avoid passing private keys as strings on the command line. How would you feel about making these file-based arguments? We have something for this already in the storage node, which accepts either a json file, or a PEM file: https://github.com/storacha/storage/blob/main/cmd/start.go#L335